core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts

```diff
@@ -447,7 +459,9 @@ export class ServiceMetricsComponent implements OnInit, AfterViewInit {
         break;
       case 'website':
         let websiteAssets = _(this.queryDataRaw.assets).map('type').uniq().value();
-        this.filters.addField('Filter By:', 'ASSET', websiteAssets, 'select', null, 'cloudfront');
+        if (this.provider != 'azure'){
```

Comment:
Will this work if we add another provider. Should we just say `this.provider == 'aws'` instead?

Reply:
My Bad. Changed to provider == 'aws'